.travis.yml

```diff
@@ -9,8 +9,18 @@ env:
   - TOXENV=py34
   - TOXENV=py36
 
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y swig
+
 before_script:
   - pip install -U pip wheel tox
+  - sudo wget https://dist.opendnssec.org/source/softhsm-2.3.0.tar.gz
+  - sudo tar -xzf softhsm-2.3.0.tar.gz
+  - cd softhsm-2.3.0
+  - sudo ./configure
+  - sudo make
+  - sudo make install
```

Comment:
Yes, sudo is needed for all the commands!
Sure, I would keep that in mind while resolving the conflict!

Reply:
This is most likely *not* the case.  You need sudo because you wget'ed with sudo and then untar'd with sudo, so the resulting files are owned by root.  The only command that should need sudo here is probably the make install